pallets/xyk/src/tests.rs

```diff
@@ -904,7 +906,7 @@ fn multiswap_sell_bad_atomic_swap_charges_fee_W() {
 
 		assert_ok!(XykStorage::multiswap_sell_asset(
 			RuntimeOrigin::signed(TRADER_ID),
-			vec![1, 2, 3, 6, 5],
+			vec![1, 2, 3, 4, 5],
```

Comment:
This was probably an old bug in our tests that went unnoticed until this added validation